PYI-297: Add dependencies for api-gateway integration

kerrr opened the pull request:
## Proposed changes

### What changed

Add dependencies for the `aws_api_gateway_integration_response` resource.

### Why did it change

Terraforming the api gateway from scratch revealed a dependency issue. 

### Issue tracking
<!-- List any related Jira tickets or GitHub issues -->
<!-- List any related ADRs or RFCs -->
<!-- Delete/copy as appropriate -->

- [PYI-297](https://govukverify.atlassian.net/browse/PYI-297)

## Checklists

### Environment variables or secrets

<!-- Delete if changes DO include new environment variables or secrets -->
- [ x ] No environment variables or secrets were added or changed

merlinc reviewed (commented):
LGTM

merlinc reviewed (approved):
LGTM for sure this time!

kerrr committed:
PYI-297: Add dependencies for api-gateway integration

Terraforming the api gateway from scratch revealed a dependency issue. Add dependencies for the `aws_api_gateway_integration_response` resource.
